docker/Dockerfile.ort-default-stable-torch190-onnxruntime181-cu102-cudnn7-devel-ubuntu18.04

```diff
@@ -23,7 +23,7 @@ RUN update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
 RUN pip install --upgrade pip
 
 # PyTorch
-RUN pip install onnx ninja
+RUN pip install onnx==1.9.0 ninja
```

Comment:
Do we need to explicitly install the dependencies? I thought they were included in the requirements file?

Reply:
but this is docker file. it shall be standalone and does not use requirement.txt